Lib/tkinter/font.py

```diff
@@ -100,6 +100,9 @@ def __init__(self, root=None, font=None, name=None, exists=False,
     def __str__(self):
         return self.name
 
+    def __repr__(self):
+        return f"{self.__class__.__module__}.{self.__class__.__name__} object '{self.name}'"
```

Comment:
And instead of `'{self.name}'` it is better to use `{self.name!r}`.

Reply:
Oh, cool! I didn't know about this feature. Yeah, !r looks much better :)